dataminer/Functions/Video_Thumbnails/Configuring_Video_Thumbnails/Adding_parameters_to_the_video_URL.md

```diff
@@ -0,0 +1,112 @@
+
+    - From DataMiner 10.2.0 [CU1]/10.2.4 onwards, HTML5 video thumbnails are muted by default. From DataMiner 10.4.0 [CU11]/10.5.2 onwards<!--RN 41597-->, you can unmute them by clicking the sound icon in the lower-right corner. When unmuted, the volume is automatically set to 100%. In earlier versions, unmuting required hovering over the sound icon and using a slider to adjust the volume.
+
+  - HLS (HTTPS Live Streaming): `type=HTML5-HLS`
+
+    - From DataMiner web 10.4.0 [CU10]/10.5.1 onwards<!--RN 41407-->, if the HLS stream has multiple audio tracks, you can select your preferred audio track using the dropdown menu in the top-right corner of the video thumbnail.
+
+    - All HLS resources must be delivered with CORS headers that permit GET requests. For more information, see <https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS>.
+
+    - If you access a video thumbnail player that is using HTTPS, the media must also be served over HTTPS.
+
+    > [!TIP]
+    > For more information on HLS, see <https://github.com/video-dev/hls.js/>
+
+  - Image: `type=Generic%20Images`
+
+    - The following image formats are supported: .png, .jpg, and .bmp.
+
+    - If you want to display an image located on a DMA, place the image in the DMA's `C:\Skyline DataMiner\Webpages\Public\` folder (or one of its subfolders, e.g. `C:\Skyline DataMiner\Webpages\Public\MyImages\`).
+
+      > [!TIP]
+      > If you get an "Invalid path" error, open the file `C:\Skyline DataMiner\Webpages\VideoThumbnails\Web.config`, and check whether the image folder (e.g. */Public/MyImages/*) has been added to the *ExtraAllowedPaths* key.
+      >
+      > See also: [Allowed paths in case of connection via DataMiner proxy](xref:Allowed_paths_connection_DataMiner_proxy).
+
+    - Add a `refresh=` parameter to the video URL to specify the refresh interval (in milliseconds). See [Optional parameters](#optional-parameters).
+
+    - By default, a thumbnail of type *Generic Images* always uses the DMA as a proxy. However, you can add an extra URL parameter, "proxy", to override this behavior. For example: `http://<DMA>/VideoThumbnails/video.htm?type=Generic%20Images&source=<IMG URL>&proxy=false`
+
+    - Both HTTP and HTTPS are supported.
+
+  - VLC: `type=Generic%20VLC`
+
+    - To display video thumbnails with the VLC plugin in the Cube desktop app, use the 64-bit version of VLC.
+
+- `source=`: The URL of the video/image.
+
+## Optional parameters
+
+- `referer=` (Generic VLC): The VLC component of video thumbnails sends a referer HTTP header when requesting the source URL. The referer URL is by default the URL of the DMA, but you can change it by specifying the parameter "referer=" in the URL of the video thumbnail.
+
+  Example: `#http://<DMA>/VideoThumbnails/video.htm?type=VLC&source=http%3A%2F%2Fclips.vorwaerts-gmbh.de%2Fbig_buck_bunny.mp4&referer=http%3A%2F%2Fsome%2Freferer%2F.`
```

Comment:
It should be \<DMAIP\> in all these examples, instead of \<DMA\> (this placeholder doesn't exist).

The naming is confusing, because \<DMAIP\> can resolve into a hostname. See https://docs.dataminer.services/dataminer/Operator_guide/Visio/reference/Placeholders_for_variables_in_shape_data_values.html#dmaip